features/bootstrap/FeatureContext.php

```diff
@@ -60,6 +60,7 @@ public static function cleanTestFolders()
         if (is_dir($dir = sys_get_temp_dir() . DIRECTORY_SEPARATOR . 'behat')) {
             self::clearDirectory($dir);
         }
+        self::clearDirectory('tests/Fixtures/temp', false);
```

Comment:
why do we need `tests/Fixtures/temp` ? Can temporary files be kept in `sys_get_temp_dir()` ?

Reply:
And if we keep temp files inside `tests/Fixtures`, we really need a `.gitignore` rule to avoid committing them by mistake

Reply:
I did this because we had some steps where you had to specify the path of the file to check and I could not use a temp folder, but then I realised that I could add some new steps that used the temp folder, updated